### PR TITLE
Add script for smoke tests

### PR DESCRIPTION
**Pull request by mheilman**
This adds a script for running some simple tests that hit the live Civis platform.

### CODE REVIEW

File: tools/smoke_tests.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    logger.info('Testing reading from redshift...')
+    sql = "SELECT * FROM datascience.iris"
+    df1 = civis.io.read_civis_sql(
+        sql=sql, database=database, use_pandas=True, client=client)
+    df2 = civis.io.read_civis(
+        table="datascience.iris", database=database, use_pandas=True,
+        client=client
+    )
+    assert df1.shape == (150, 6)
+    pd.testing.assert_frame_equal(df1, df2)
```
Comment (keithing) [inline tools/smoke_tests.py]: is it possible the ordering to the two tables comes back differently from the database and this results in `false`?

Comment (mheilman) [reply tools/smoke_tests.py]: oh, good point.  i haven't seen that, but i'll add an order by or sort after downloading

Comment (mheilman) [reply tools/smoke_tests.py]: @patr1ckm, the r client test does the same thing [here](https://github.com/civisanalytics/civis-r/blob/master/tools/integration_tests/smoke_test.R#L29), right?

**Pull request merged**